app/models/gobierto_plans/node.rb

```diff
@@ -14,7 +14,21 @@ class Node < ApplicationRecord
     scope :with_status, ->(status) { with_status_translation(status) }
     scope :with_category, ->(category) { where(gplan_categories_nodes: { category_id: GobiertoCommon::Term.find(category).last_descendants }) }
     scope :with_progress, ->(progress) { where("progress > ? AND progress <= ?", *progress.split(" - ")) }
-    scope :with_interval, ->(interval) { where("starts_at >= ? AND ends_at <= ?", *interval.split(",").map { |date| Date.parse(date) }) }
+    scope :with_interval, lambda { |interval|
+      dates = interval.split(",").map do |date|
+        begin
+          Date.parse(date)
+        rescue ArgumentError
+          nil
+        end
+      end
+
+      dates = [nil, nil].zip(dates).map { |date| date.compact.first }
+
+      where("starts_at >= ? AND ends_at <= ?", *dates)
```

Comment:
I think I'd change this AND by an OR. This way, when I search projects between two dates I'll get all the projects active in those dates. If you use an AND you'll get all the projects that start or end inside that range, which I think is more difficult to understand from the users side.